sympy/printing/latex.py

```diff
@@ -546,7 +546,7 @@ def convert_args(args):
         # XXX: _print_Pow calls this routine with instances of Pow...
         if isinstance(expr, Mul):
             args = expr.args
-            if args[0] is S.One or any(isinstance(arg, Number) for arg in args[1:]):
+            if args[0] is S.One or any(isinstance(arg, Number) for arg in args[1:]) and args[0].is_number:
```

Comment:
Can you explain what this change is for?

I don't think `is_number` is the right thing to check here.

Reply:
I added that to make sure latex(Mul(x + y, Rational(1, 2), evaluate=False)) outputs '\\frac{x + y}{2}' which before is returning a different output '\\left(x + y\\right) \\frac{1}{2}' due to checking for second arg as a number.